### PR TITLE
Makefile fixes

### PR DESCRIPTION
**Pull request by ltratt**
[Needs #184 to be merged first.]

Some simple tweaks to the Makefile that make it work better IMHO.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,7 +34,7 @@ build-benchmarks: build-krun
 		env LD_LIBRARY_PATH=${GCC_LIB_DIR} \
 		CC=${CC} JAVAC=${JAVAC} ${MAKE}
 
-build-krun:
+build-krun: build-vms
```
Comment (vext01) [inline Makefile:37]: At first I didn't understand, but I think I do now. libkruntime.so depends upon our libjava, right?

**Comment by vext01** [comment]
Looks ok bar one comment.

File: Makefile (hunk 1 above)
Comment (ltratt) [reply Makefile:37]: And our gcc.

Comment (vext01) [reply Makefile:37]: ah yeah.

**Comment by ltratt** [comment]
Rebasing this one too (I assume your comment doesn't need addressing?).

**Comment by vext01** [comment]
Any squashing. Probably not, but I should ask.

**Comment by ltratt** [comment]
I think the commits are best left separate.

Ready for merging.

**Review by vext01 — APPROVED** [review]

**Pull request merged**